packages/docs/src/components/Modal/Modal.stories.mdx

```diff
@@ -0,0 +1,83 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import * as Stories from './examples';
+
+<Meta title="Components|Modal" />
+
+# Modal
+
+The Modal component is a basic way to present content above an enclosing view.
+Modals may be nested within other Modals.
+
+## Props
+
+| Name                      | Type           | Default |
+| ------------------------- | -------------- | ------- |
+| children                  | ?any           | |
+| visible                   | ?boolean       | false |
```

Comment:
`visible` should be `true` by default

https://github.com/facebook/react-native/blob/master/Libraries/Modal/Modal.js#L149